### DISCUSSION TITLE
Forgot the user's password

### DISCUSSION THREAD

**Discussion by Joackk** [dani-garcia/vaultwarden#2858] [Q&A] [answered]
I accidentally forgot my master password, what should I do? There is no option to change the user password on the administration page, can I still change the password for this account? Or can I re-export a copy of the data?

**Comment by KnightTim** [comment, marked as answer]
https://bitwarden.com/help/forgot-master-password/ 

TLDR: The data in your vault is encrypted with the master password and cannot be recovered without it. If you are still logged in somewhere write down your passwords.  Then you can delete your account and start over.

**Reply by BlackDex** [reply]
A small side note, on a mobile client with internet for example you can export your personal vault items to a json or csv.
